sgp4/functions.py

```diff
@@ -8,7 +8,7 @@
 def jday(year, mon, day, hr, minute, sec):
      """Return two floats that, when added, produce the specified Julian date.
 
-     The first float returned gives the date, while the second float
+     The first float returned (jd) gives the date, while the second float (fr)
```

Comment:
Nice: I love the idea of, even in the docstring (and not just in the main docs), telling users good names for the two return values.

But I'm not sure we should do it mid-paragraph? Unless we decide to return a namedtuple, these won't be names that programmers can use. If they try something like:
```
result = jday(...)
print(result.jd)
```
— then they'll get a `NameError`. So I suspect that advertising the names might be confusing, since by default Python tuples arrive with no fixed names for their values.

**However,** I do love suggesting good names to developers, it's a great practice, it leads to more readable user code and more consistent code, so I completely agree with your goal here. So here's another way to go about it: add to the docstring a little sample line of code showing how to call it:
```
You can call this like:

jd, fr = jday(...)
```
That way, we both teach users about the best names to use, make their code more consistent, but also make it clear _how_ the values are going to get those names in their code: not because we are setting those names, but because the users themselves will set those names voluntarily through tuple unpacking. Otherwise, novice Python programmers might not know exactly how to give those two names to the two return values.

Reply:
You've already called them jd, fr -- I was making the documentation more explicit than requiring the user to mentally equate first = jd, second = jr